src/Resources/config/cors.yml

```diff
@@ -0,0 +1,3 @@
+nelmio_cors:
+    paths:
+        '/': # Nonsense configuration as the nelmio compiler pass will abort if not present
```

Comment:
Why don't we use `'^/'` as in the [example configuration](https://github.com/nelmio/NelmioCorsBundle#configuration)?